File: Python/Iniciante/1015.py
import math
l1 = input().split(" ")
l2 = input().split(" ")
a1, b1 = l1
a2, b2 = l2

x1 = float(a1)
y1 = float(b1)
x2 = float(a2)
y2 = float(b2)

dist = math.sqrt((x2-x1)*(x2-x1) + (y2-y1)*(y2-y1))

print("%.4f" %(dist))

File: C/Iniciante/1015.c
#include <stdio.h>
#include <math.h>

int main (void)
{
	double X1, Y1, X2, Y2, RES;
	scanf ("%lf %lf", &X1, &Y1);
	scanf ("%lf %lf", &X2, &Y2);
	RES = sqrt((X2-X1)*(X2-X1)+(Y2-Y1)*(Y2-Y1));
	printf ("%.4f\n", RES);

	return 0;
}